docker/docker-compose.yml

```diff
@@ -67,7 +67,7 @@ services:
       - RABBITMQ_PASSWORD=${RABBITMQ_PASSWORD}
       - AGENT_ADMIN_URL=http://vcr-agent:${AGENT_ADMIN_PORT}
       - AGENT_ADMIN_API_KEY=${AGENT_ADMIN_API_KEY}
-      - API_VERSION_ROUTING_MIDDLEWARE=${API_VERSION_ROUTING_MIDDLEWARE}
+      - API_VERSION_ROUTING_MIDDLEWARE=${API_VERSION_ROUTING_MIDDLEWARE:-vcr_server.middleware.routing.HTTPHeaderRoutingMiddleware}
```

Comment:
Any reason this was changed?

Reply:
this was changed by the merge pull request

Reply:
If you are merging into `master` and the change is there, Git should not be displaying a diff for it (unless there is a glitch).
At the very least this will cause a merge conflict since the git tree will have the same change from two different git histories, so it's worth checking...

Reply:
fixed it, there was something weird going on and my branch fell a couple commits behind. I did another merge pull and that sorted things out